src/main/java/org/broadinstitute/hellbender/engine/filters/OverclippedReadFilter.java

```diff
@@ -14,9 +14,9 @@
  *     This filter is intended to filter out reads that are potentially from foreign organisms.
  *     From experience with sequencing of human DNA we have found cases of contamination by bacterial
  *     organisms; the symptoms of such contamination are a class of reads with only a small number
- *     of aligned bases and additionally many soft-clipped bases.  This filter is intended
- *     to remove such reads. Consecutive soft-clipped blocks are treated as a single block
- * </p>
+ *     of aligned bases and additionally many soft-clipped bases. This filter is intended
+ *     to remove such reads.
```

Comment:
So the hardclips to not count towards softclips but they do indirectly reduce the number of matches. It would be great to include your (1) so user's know what counts towards alignment.